docs/examples/nginx-ingress.md

```diff
@@ -239,4 +239,4 @@ For more information about NGINX Ingress Controller installation, take a look at
 
 ## Alternative examples for Ingress Controllers on k0s
 
-[Traefik Ingress](https://docs.k0sproject.io/main/examples/traefik-ingress/)
+[Traefik Ingress](https://docs.k0sproject.io/stable/examples/traefik-ingress/)
```

Comment:
I'd rather make this "relative" and not pin it to specific version of docs

Reply:
I think that will require using `../traefik-ingress`

Reply:
https://docs.k0sproject.io/v1.23.3+k0s.1/examples/nginx-ingress/../traefik-ingress seems to take the browser to the right location.